doc/smart-switch/pmon/static-ip-assignment.md

```diff
@@ -0,0 +1,51 @@
+# Midplane Interface 
+
+A typical modular chassis includes a midplane-interface to interconnect the Supervisor & line-cards. The same design has been extended in case of a SmartSwitch. The mnic ethernet interface over PCIe which is the midplane-interface, interconnect the Switch Host and the DPUs.
+
+* When DPU card (SLED) or the Supervisor boots and as part of its initialization, midplane interface gets initialized.
+* Slot number is used in assigning an IP address to these interfaces.
+
+## 1.    DPU static IP address allocation
```

Comment:
our POR is DHCP, right?

Reply:
Yes is the answer, but this is outdated.